WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
It badly needed an overhaul. It had so many unbalanced crap and, worse, so many terrible names. It was a mostly broken, crappy file that you refused to fix. I did your job for you.